### PR TITLE
Provision to use alternate repos for helpernode and playbooks

### PR DESCRIPTION
**Pull request by Prajyot-Parab**
Signed-off-by: Prajyot Parab <prajyot.parab@ibm.com>

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -208,12 +208,24 @@ variable "installer_log_level" {
     default = "info"
 }
 
+variable "helpernode_repo" {
+    description = "Set the repo URL for using ocp4-helpernode"
+    # Repo for running ocp4 installations steps.
```
Comment (yussufsh) [inline variables.tf]: Update the comment please

**Review by yussufsh — CHANGES_REQUESTED** [review]

**Comment by bpradipt** [comment]
@Prajyot-Parab can you please tag this to an issue/feature request

**Comment by Prajyot-Parab** [comment]
> @Prajyot-Parab can you please tag this to an issue/feature request

@bpradipt Done

**Review by yussufsh — APPROVED** [review]

**Comment by bpradipt** [comment]
@Prajyot-Parab since we are adding support for alternate repos, I would like to see an option to use zip or tar.gz of the helpernode and playbook. Would this make sense to incorporate into this patch  - Support for alternate repos and format ?

**Comment by Prajyot-Parab** [comment]
> @Prajyot-Parab since we are adding support for alternate repos, I would like to see an option to use zip or tar.gz of the helpernode and playbook. Would this make sense to incorporate into this patch - Support for alternate repos and format ?

@bpradipt I can explore on that and push it as a separate patch.

**Comment by bpradipt** [comment]
ok @Prajyot-Parab please explore that as it will be critical for completely disconnected install.

**Review by bpradipt — APPROVED** [review]
/lgtm
Thanks @Prajyot-Parab

**Comment by yussufsh** [comment]
> 
> 
> @Prajyot-Parab since we are adding support for alternate repos, I would like to see an option to use zip or tar.gz of the helpernode and playbook. Would this make sense to incorporate into this patch - Support for alternate repos and format ?

With tar/gz/zip there are various validations for type of archive and directory structure. It is complex and need to create an issue for this.

**Comment by yussufsh** [comment]
We can work with this one https://github.com/ocp-power-automation/ocp4-upi-powervm/issues/30

**Pull request merged**